kube-derive/src/lib.rs

```diff
@@ -111,6 +111,22 @@ mod custom_resource;
 /// Adding `#[kube(derive = "PartialEq")]` is required if you want your generated
 /// top level type to be able to `#[derive(PartialEq)]`
 ///
+/// ### `#[kube(schema_mode = "mode")]`
```

Comment:
Can we change the name of this to `#[kube(schema = enumname)]`? There's no clashing long names here, and `_mode` doesn't add a lot of context.

Reply:
That was mostly done to avoid confusion with `schemars`. Maybe another approach would be to move `schemars` and co into an `overrides()` subattribute, or something?

Reply:
Would that look like something like `#[kube(overrides(schema(..))]`?

I'm not sure overrides here is right, because we are not really overriding any `schemars` functionality, just using it differently.

Reply:
I was thinking something like `#[kube(overrides(kube_core = ..., schemars = ...), schema = "custom")]`

Reply:
Ah, I see now. That is a pretty nice idea. Encapsulates the common path overrides in one struct.  Does `darling` let you nest structs while nesting `#[darling(attributes..)]` on these sub structs?

Reply:
I *think* you could do this by deriving `FromMeta`, but I haven't actually tried that.

Reply:
92090b1f121f6c2ce98fb145eeb0023fdc7fbc06